src/Host.cpp

```diff
@@ -1217,7 +1220,7 @@ void Host::readPackageConfig(const QString& luaConfig, QString& packageName)
             qDebug() << reason.c_str() << " in config.lua:" << e.c_str();
         }
         // should print error to main display
-        QString msg = QString("%1 in config.lua: %2\n").arg(reason.c_str(), e.c_str());
+        QString msg = QStringLiteral("%1 in config.lua: %2\n").arg(reason.c_str(), e.c_str());
```

Comment:
Does there need to be a `.` for the sentence before the line-feed in the `"%1 in config.lua: %2\n"`?

Reply:
I don't think so.